src/explore-education-statistics-admin/src/pages/admin-dashboard/components/PublicationSummary.tsx

```diff
@@ -57,41 +67,19 @@ const PublicationSummary = ({
         </SummaryListItem>
         <SummaryListItem term="Releases" smallKey>
           <ul className="govuk-list dfe-admin">
-            {publication.releases.map(release => (
-              <li key={release.id}>
-                <ReleaseSummary
-                  release={release}
-                  actions={
-                    <>
-                      <ButtonLink
-                        to={summaryRoute.generateLink({
-                          publicationId: publication.id,
-                          releaseId: release.id,
-                        })}
-                        testId={formatTestId(
-                          `Edit release link for ${
-                            publication.title
-                          }, ${getReleaseSummaryLabel(release)}`,
-                        )}
-                      >
-                        {release.permissions.canUpdateRelease
-                          ? 'Edit this release'
-                          : 'View this release'}
-                      </ButtonLink>
-                      {showAmendmentButton() &&
-                        release.permissions.canMakeAmendmentOfRelease && (
-                          <Button
-                            className="govuk-button--secondary govuk-!-margin-left-4"
-                            onClick={() => setAmendReleaseId(release.id)}
-                          >
-                            Amend this release
-                          </Button>
-                        )}
-                    </>
-                  }
-                />
-              </li>
-            ))}
+            {publication.releases
+              .filter(noAmendmentInProgressFilter)
+              .map(release => (
+                <li key={release.id}>
+                  <NonScheduledReleaseSummary
+                    onClickAmendRelease={
+                      showAmendmentButton() ? setAmendReleaseId : undefined
```

Comment:
`showAmendmentButton()` is always false, why are we even bothering with this?

Reply:
Basically a quick way of allowing us to continue working on this feature whilst not enabling it on the front end until we're ready to roll it out.